samigo/samigo-services/src/java/org/sakaiproject/tool/assessment/facade/PublishedAssessmentFacadeQueries.java

```diff
@@ -346,6 +347,7 @@ public Set preparePublishedSectionMetaDataSet(
 					sectionMetaData.getEntry());
 			h.add(publishedSectionMetaData);
 		}
+		h.add(new PublishedSectionMetaData(publishedSection, SectionDataIfc.RANDOMIZATION_SEED, String.valueOf(UUID.randomUUID().hashCode())));
```

Comment:
Please include this comment:

// Persist the random seed in the section to use it and preserve the order.